sxt/cbindings/backend/cpu_backend.h

```diff
@@ -36,7 +36,7 @@ struct element_p3;
 
 namespace sxt::cg1t {
 class compressed_element;
-struct element_p3;
+struct element_p2;
```

Comment:
You should be able to drop these forward declarations since they should be defined in the included backend.h header

Reply:
Good call, I removed all the forward declarations in `cpu_ backend.h` and `gpu_backend.h`.